tls_codec/src/lib.rs

```diff
@@ -288,13 +302,14 @@ impl From<U24> for usize {
 impl TryFrom<usize> for U24 {
     type Error = Error;
 
-    fn try_from(value: usize) -> Result<Self, Self::Error> {
+    fn try_from(value: usize) -> Result<Self, Error> {
```

Comment:
Can't hax handle this? It would be better to not having to keep these two types in sync manually.